src/components/Receive.jsx

```diff
@@ -12,6 +12,7 @@ import Sprite from './Sprite'
 export default function Receive({ currentWallet }) {
   const location = useLocation()
   const settings = useSettings()
+  const inputRef = useRef()
```

Comment:
In order for others not to guess or scroll around too much, maybe we can come up with a more descriptive name. Instead of `inputRef` maybe `addressCopyFallbackInputRef`?